content/governance.md

```diff
@@ -0,0 +1,175 @@
+
+Votes may take the form of a single proposal, with the option to vote yes or no; or the form of multiple alternatives.
+
+A vote on a single proposal is considered successful if at least two thirds of those eligible to vote vote in favor.
+
+If there are multiple alternatives, members may vote for one or more alternative, or vote “no” to object to all alternatives. A vote on multiple alternatives is considered decided in favor of one alternative if it has received the most votes in favor, and a vote from at least two thirds of those eligible to vote. Should no alternative reach this quorum, another vote on a reduced number of options may be called separately.
+
+## FAQ
+
+This section is informational. In case of disagreement, the rules above overrule any FAQ.
+
+### How do I propose a decision?
+
+Send an email to [the developer mailing list](devs) with your motion. If there is no objection within a reasonable amount of time, consider the decision made. If there are objections and no consensus can be found, a vote may be called by a team member.
+
+### How do I become a team member?
+
+To become an official team member, you should make ongoing contributions to one or more project(s) for at least three months. At that point, a team member (typically a maintainer of the project) may propose you for membership. The discussion about this will be held in private, and you will be informed privately when a decision has been made. A possible, but not required, graduation path is to become a maintainer first.
+
+Should the decision be in favor, your new membership will also be announced on the [developers mailing list][devs].
+
+### How do I add a project?
+
+As a team member, propose the new project on the [developer mailing list][devs]. If nobody objects, create the project in the Github organization. Add at least a README.md explaining the goal of the project, and a MAINTAINERS.md with the maintainers of the project (at this point, this probably means you).
+
+### How do I archive or remove a project?
+
+Send an email to the [developer mailing list][devs] proposing the retirement of a project. If nobody objects, move it to the prometheus-junkyard Github organization.
+
+### How do I remove an inactive maintainer?
+
+A maintainer may resign by notifying the [team mailing list][team]. A maintainer with no project activity for a year will be treated as if they had resigned. If there is an urgent need to replace a maintainer, discuss this on the [team mailing list][team].
+
+### How do I remove a team member?
+
+Team members may resign by notifying the [team mailing list][team]. If you think a team member should be removed against their will, propose this to the [team mailing list][team]. Discussions will be held there in private.
+
+[team]: https://groups.google.com/forum/#!forum/prometheus-team
```

Comment:
Necessary to link this since the vast majority of readers can't access it?

Reply:
Hmm, good question. Linking it makes it unambiguous which mailing list is referenced. When a non-member opens it, can they see that it exists?

Reply:
In incognito window:

```
Prometheus Team
You must be signed in as a member of this group to view and participate in it.
Sign in to view this group
```

Reply:
I think then it's fair to link it. The point is not that there is something to read there, but to be very specific about the mailing list we refer to. Also, team members _can_ use this link to refer back to it.

Reply:
(I'm pretty sure there won't be _that_ many readers besides those deeply involved in the project)